.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+<!-- Add a short description of what your PR does here -->
+
+## Changes
+
+-
+
+## Screenshots
+
+<!-- (prefer animated gif) -->
+
+## Checklist
+
+- [ ] tested locally
+- [ ] updated the docs
+
+Fixes #
```

Comment:
Not part of this PR.. Maybe we should skip that.

Reply:
To be honest, I don't really like these templates, if we see that is needed in the future we could reconsider.

Reply:
Okay, I'll remove that